src/dlr.cc

```diff
@@ -54,6 +54,26 @@ extern "C" int GetDLRInputType(DLRModelHandle* handle, int index,
   API_END();
 }
 
+extern "C" int GetDLRInputShape(DLRModelHandle* handle, int index,
+                                 int64_t* shape) {
+  API_BEGIN();
+  DLRModel* model = static_cast<DLRModel*>(*handle);
+  CHECK(model != nullptr) << "model is nullptr, create it first";
+  std::vector<int64_t> input_shape = model->GetInputShape(index);
```

Comment:
Make `input_shape` a reference

Reply:
GetInputShape returns a reference.

Reply:
Yeah, so we need to do ` std::vector<int64_t>& input_shape = model->GetInputShape(index);` so that input_shape  is also a reference. Otherwise, the current code will create an unnecessary local copy of the shape.

Reply:
This change leads to `/home/ubuntu/TVM/neo-ai-dlr/src/dlr_treelite.cc:122:46: error: binding reference of type ‘std::vector<long int>&’ to ‘const std::vector<long int>’ discards qualifiers`